tests/formulas/tokenizer.test.ts

```diff
@@ -157,6 +157,14 @@ describe("tokenizer", () => {
       { type: "OPERATOR", value: "=" },
       { type: "SYMBOL", value: "'a '' b'!A1" },
     ]);
+    expect(tokenize("=1name!A1")).toEqual([
```

Comment:
you could add a test where the sheet name is only numbers `123!A1`

Reply:
done, and it works !!  That being said, I really don't like the way I fix this, but I think it's better to ensure it's not detected as a number rather than let is pass and try to recompose it with `mergeSymbolsIntoRanges`  wdyt ?

Reply:
>  I really don't like the way I fix this

Neither do I, our tokenizer is too smart by trying to do too many things at once

> it's better to ensure it's not detected as a number rather than let is pass and try to recompose it with mergeSymbolsIntoRanges wdyt

definitely, especially in 15.0